internal/cfgplugins/interface.go

```diff
@@ -725,3 +725,23 @@ func DeleteAggregate(t *testing.T, dut *ondatra.DUTDevice, aggID string, dutAggP
 		gnmi.Delete(t, dut, gnmi.OC().Interface(port.Name()).Ethernet().AggregateId().Config())
 	}
 }
+
+// SetupStaticAggregateAtomically sets up the static aggregate interface atomically.
+func SetupStaticAggregateAtomically(t *testing.T, dut *ondatra.DUTDevice, aggPorts []*ondatra.Port, aggID string) {
+	t.Helper()
+	d := &oc.Root{}
+	agg := d.GetOrCreateInterface(aggID)
+	agg.Type = oc.IETFInterfaces_InterfaceType_ieee8023adLag
+	agg.GetOrCreateAggregation().LagType = oc.IfAggregate_AggregationType_STATIC
+
+	for _, port := range aggPorts {
+		i := d.GetOrCreateInterface(port.Name())
+		i.GetOrCreateEthernet().AggregateId = ygot.String(aggID)
+		i.Type = oc.IETFInterfaces_InterfaceType_ethernetCsmacd
+
+		if deviations.InterfaceEnabled(dut) {
+			i.Enabled = ygot.Bool(true)
+		}
+	}
+	gnmi.Update(t, dut, gnmi.OC().Config(), d)
```

Comment:
this function should include a gnmi Batch in the parameters and return the same.  See this example for the style:
https://github.com/openconfig/featureprofiles/blob/bc105a443b44d862c70e91112708b5a339c71ae5/internal/cfgplugins/sflow.go#L33-L55